pkgs/by-name/st/stormlib/package.nix

```diff
@@ -1,41 +1,60 @@
-{ lib, stdenv, fetchFromGitHub, cmake, bzip2, libtomcrypt, zlib, darwin }:
+{
+  lib,
```

Comment:
I think the common format is still:
```
{ lib
, bzip2
, ...
}
```

Reply:
nixfmt has introduced a new style as per rfc 166.